### PR TITLE
Doc/trello guide

### PR DESCRIPTION
**Pull request by timmydzeko**
Trello guide created in docs/training. The main points have been covered. Still need to add images and SplashKit Modules board example. Looking for feedback on whether anything needs to be added or worded better.

### CODE REVIEW

**Comment by ghost** [comment]
Content looks good Tim! 
Are you able to run the Prettier Extension on the code. This is available as part of VS Code.
We have a new feature within this repo which will block any merges due to formatting - this will fix this check

**Comment by ghost** [comment]
@tancnle Could you assist:
we are having trouble identifying why prettier is failing - it has been run in VS Code by both machines

**Review by ginge270 — COMMENTED** [review]
Great Job!

**Comment by tancnle** [comment]
@talzid Interesting. Do you see any errors when running the check command locally on this branch?

```sh
npm run format:check
```

I have checked the auto-format on VS Code in my Mac and it did picked up these issues 🤔

**Review by ginge270 — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -1,35 +1,7 @@
...
-      },
-      "engines": {
-        "node": ">=10.13.0"
-      },
-      "funding": {
-        "url": "https://github.com/prettier/prettier?sponsor=1"
-      }
-    }
-  },
+  "lockfileVersion": 1,
```
Comment (tancnle) [inline package-lock.json]: **issue:** This seems like an unintended change. We should use the latest lockfile version `2`. Could we make sure we use the correct version of node and npm, that is `node >=16.14.2` and `npm ^8.5.0`? To check, we can run

```sh
node --version
npm --version
```

Comment (ghost) [reply package-lock.json]: https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl

This was very helpful to update my versions. However, we shall see how successful it is

**Comment by timmydzeko** [comment]
@talzid @tancnle I figured out my issue with Prettier. I had just opened the file direct from where I cloned the repository. I needed to first create a workspace within my .vscode directory, and then add the docs folder from the repository. It was looking for the Prettier extension outside of VSCode, which will always fail.

**Comment by ghost** [comment]
Nice trouble shooting @timmydzeko 
I am about to fix the errors I incidentally made in fixing the formatting. Will publish and merge in a minute. Please do not commit anything in the meantime 👍

**Review by tancnle — APPROVED** [review]
Great work and collaboration @timmydzeko and @talzid ❤️ The PR looks bonza to me 🚀

**Pull request merged**